src/Stratis.Bitcoin.Features.Miner/Controllers/MiningController.cs

```diff
@@ -20,40 +20,17 @@ namespace Stratis.Bitcoin.Features.Miner.Controllers
     [Route("api/[controller]")]
     public class MiningController : Controller
     {
-        /// <summary>Instance logger.</summary>
+        private const string exceptionOccurredMessage = "Exception occurred: {0}";
+        private readonly IFullNode fullNode;
         private readonly ILogger logger;
 
-        /// <summary>PoW miner.</summary>
-        private readonly IPowMining powMining;
-
-        /// <summary>Wallet manager.</summary>
-        private readonly IWalletManager walletManager;
```

Comment:
Wait why remove the dependencies?
This is the opposite of what we want using `this.fullNode.NodeService<IWalletManager>()` is bad practice.

Reply:
OK I have reverted this 👍